src/mixins/common-mixin.ts

```diff
@@ -6,25 +6,6 @@ import { omit, isEqual } from 'lodash'
  */
 @Component({})
 export default class CommonMixin extends Vue {
-  /**
-   * This is an example mixin that will return a msg.
-   *
-   * @param msg The msg to return.
-   */
-  sendMsg (msg: string): string {
-    return msg ? `${msg} - msg created by sendMsg mixin` : 'Error, no Message'
-  }
-
-  /**
-   * Removes the specified property from an object.
-   *
-   * @param baseObj The base object.
-   * @param prop The property to be removed.
-   */
-  omitProp (baseObj: object, prop: Array<string>): object {
```

Comment:
This kinda did nothing :P

Reply:
hmm, not sure why you say that. It literally does exactly what it says. You pass it a string that references a specific prop name and omits it from the comparison?

Reply:
It isn't widely used in Edit Ui, though i can see some instances in `OfficeAddresses`, but it might not be working there because the prop being passed doesn't exist on the objects of comparison in the first place.. according to the interface anyway.

Reply:
It's just a wrapper for `omit` isn't it?

Reply:
Yea, seems so! Scratching my head now as to why it was done in the first place 😆 
👍